docker/scylla-sct/Dockerfile

```diff
@@ -30,3 +30,7 @@ ADD etc/sshd.conf /etc/supervisord.conf.d/sshd.conf
 ADD etc/collectd.conf /etc/supervisord.conf.d/collectd.conf
 ADD etc/scylla-manager.conf /etc/supervisord.conf.d/scylla-manager.conf
 
+# replacement of systemctl, since inside docker there is no D-Bus connection available
+RUN yum install -y wget
+RUN wget https://raw.githubusercontent.com/gdraheim/docker-systemctl-replacement/master/files/docker/systemctl.py -O /usr/bin/systemctl
```

Comment:
Can we replace wget by curl?

Reply:
Good catch, will replace by curl. Thanks.